priv/http_api/backend_swagger.yaml

```diff
@@ -0,0 +1,430 @@
+          name: XMPPHost
+          in: path
+          description: The XMPP hostname
+          required: true
+          type: string
+      description: Retrieve **paginated** session information for the given XMPP host.
+      responses:
+        200:
+          description: Retrieve session information for the given XMPP host.
+          schema:
+            title: Sessions
+            type: array
+            items:
+              title: Session
+              type: object
+              properties:
+                sid:
+                  type: string
+                jid:
+                  type: string
+                connection:
+                  type: string
+                ipAddr:
+                  type: string
+                port:
+                  type: integer
+                priority:
+                  type: integer
+                node:
+                  type: string
+                uptime:
+                  type: string
+        404:
+          description: Failure response because the given XMPP host does not exist
+  /{XMPPHost}/session/{username}:
+    parameters:
+      -
+        name: XMPPHost
+        in: path
+        description: The XMPP hostname
+        required: true
+        type: string
+      -
+        name: username
+        in: path
+        description: The username part of a JID
+        required: true
+        type: string
+    delete:
+      description: Kick a user session
+      parameters:
+        -
+          name: resource
+          in: body
+          description: The user's resource with the corresponding session to  kick
+          required: true
+          schema:
+            title: resource
+            type: string
+      responses:
+        200:
+          description: User kicked
+        404:
+          description: Failure response because the given user does not exist under this XMPP host or the XMPP host itself does not exist
+    get:
+      description: The session information for the given user.
+      responses:
+        200:
+          description: Retrieve the session information for the given user
+          schema:
+            title: Sessions
+            type: array
+            items:
+              title: Session
+              type: object
+              properties:
+                sid:
+                  type: string
+                jid:
+                  type: string
+                connection:
+                  type: string
+                ipAddr:
+                  type: string
+                port:
+                  type: integer
+                priority:
+                  type: integer
+                node:
+                  type: string
+                uptime:
+                  type: string
+        404:
+          description: Failure response because the given user does not exist under this XMPP host or the XMPP host itself does not exist
+  /cluster:
+    get:
+      description: Retrieve a list of clustered MongooseIM nodes.
```

Comment:
The implementation should look at mnesia info, and list `running` db nodes as well as `stopped`.